controllers/clusterca_to_secret.go

```diff
@@ -0,0 +1,190 @@
+package controllers
+
+import (
+	"context"
+	"encoding/hex"
+	"fmt"
+	"time"
+
+	"github.com/go-logr/logr"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	v12 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+const (
+	Namespace  = "global-trust-certificates"
+	SecretName = "cluster-ca-secret"
```

Comment:
Static secretName won't work. A portion of the certificate hash has to be appended to the name so that previous cluster-ca certs will still be around. "cluster-ca" could be the prefix then we put first or last 5 characters of the hash at the end of this name. This won't be that big of a change. Would just need to move the logic that gets the cluster-ca file and hash first before getting/creating secret.